src/QsCompiler/Transformations/CodeTransformations.cs

```diff
@@ -10,12 +10,12 @@
 using Microsoft.Quantum.QsCompiler.Transformations.FunctorGeneration;
 using Microsoft.Quantum.QsCompiler.Transformations.SearchAndReplace;
 
-namespace Microsoft.Quantum.QsCompiler
+namespace Microsoft.Quantum.QsCompiler.Transformations
```

Comment:
This renaming of the namespace is still a breaking change - wasn't that the reason that you needed to add additional usings in a couple of files?

Reply:
Reverted to old namespace. Removed added usings in ConjugationInlining.cs, FullPreEvaluation.cs, and FunctorGeneration.cs.